lib/ansible/modules/windows/win_package.py

```diff
@@ -61,6 +61,13 @@
       - Any arguments the installer needs
```

Comment:
Should there be a brief sentence around using `options` from Ansible 2.4 instead?